src/types/index.ts

```diff
@@ -243,6 +254,15 @@ export type featuredShortcutItem = {
 
 export type featuredShortcutItems = featuredShortcutItem[]
 
+export type SingleGraphQLResponse<ResponseData> = {
```

Comment:
This type is required to keep TypeScript happy in our test file for resolvers. You'll see it used a lot in `index.test.ts`